packages/chain-adapters/src/utxo/dogecoin/DogecoinChainAdapter.test.ts

```diff
@@ -459,12 +432,19 @@ describe('DogecoinChainAdapter', () => {
     })
     it('should always be coinType 3', async () => {
       for (const accountType of adapter.getSupportedAccountTypes()) {
-        const r = adapter.getBIP44Params({ accountNumber: 0, accountType })
+        const r = adapter.getBIP44Params({
+          accountNumber: 0,
+          accountType,
+          isChange: false,
+          index: 0,
```

Comment:
Do we want to be implicit for consistency?

Reply:
done ty